helm/ctdapp/templates/prometheus-rule-prod.yaml

```
@@ -0,0 +1,145 @@
+          for: 10m
+          labels:
+            severity: critical
+        - alert: FinanceNodesOutOfFunds
+          annotations:
+            description: >-
+              xDai balance of some nodes is lower than 0.1 xDai
+            summary: Nodes lacking funds
+          expr:
+          for: 10m
+          labels:
+            severity: critical
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)